test/test_playbooks/domain_vars.yml

```diff
@@ -1,6 +1,5 @@
 ---
-domain_name: "example.org"
-domain_dns_proxy: "centos7-devel.example.com"
+domain_name: "example.com"
```

Comment:
I think we used the dot org in the past to differ more explicitly from a default forklift setup (which uses `$(hostname -s).example.com` as the primary domain name). But I'm fine with either.